commands/orgs/index.js

```diff
@@ -2,35 +2,35 @@
 
 let cli = require('heroku-cli-util')
 let co = require('co')
-let _ = require('lodash')
-
-function printJSON (orgs) {
-  cli.log(JSON.stringify(orgs, null, 2))
-}
-
-function print (orgs) {
-  orgs = _.sortBy(orgs, 'name')
-  cli.table(orgs, {
-    columns: [
-      {key: 'name', label: 'Organization', format: o => cli.color.green(o)},
-      {key: 'role', label: 'Role', format: r => r}
-    ],
-    printHeader: false
-  })
-}
+let Utils = require('../../lib/utils')
 
 function * run (context, heroku) {
   let orgs = yield heroku.get('/organizations')
-  if (context.flags.json) printJSON(orgs)
-  else print(orgs)
+  let teams = orgs.filter(o => o.type === 'team')
+
+  // Filter by teams only
+  if (context.flags.teams) {
+    orgs = teams
+  } else if (context.flags.enterprise) {
+    orgs = orgs.filter(o => o.type === 'enterprise')
+  }
+
+  if (context.flags.json) Utils.printGroupsJSON(orgs)
```

Comment:
Please use curly brackets to denote this condition. https://blog.codecentric.de/en/2014/02/curly-braces/